### PR TITLE
V0.0.1 fixes

### PR DESCRIPTION
**Pull request by wivern**
Resolves various TODOs except related to SQL

### CODE REVIEW

File: service/src/main/java/org/ohdsi/gisservice/config/EncryptionConfig.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+@Configuration
+public class EncryptionConfig {
+
+	private static final Logger logger = LoggerFactory.getLogger(EncryptionConfig.class);
+
+	@Value("${jasypt.encryptor.enabled:false}")
+	private boolean encryptorEnabled;
+
+	@Bean
+	public StringEncryptor defaultStringEncryptor(Environment env) {
```
Comment (pavgra) [inline service/src/main/java/org/ohdsi/gisservice/config/EncryptionConfig.java:22]: Shouldn't we create a single config in commons and reuse across projects? I believe it is duplicated now

File: service/src/main/java/org/ohdsi/gisservice/service/EncryptionService.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	private static final String SUFFIX = ")";
+
+	private final StringEncryptor encryptor;
+
+	public EncryptionService(StringEncryptor encryptor) {
+
+		this.encryptor = encryptor;
+	}
+
+	public String decrypt(String value) {
```
Comment (pavgra) [inline service/src/main/java/org/ohdsi/gisservice/service/EncryptionService.java]: Same question about re-usage vs duplication

File: service/src/main/java/org/ohdsi/gisservice/service/SourceService.java [hunk 3]
```diff
@@ -70,9 +87,45 @@ public Source getByKey(String key) {
...
 
-    public DataSourceUnsecuredDTO getDataSourceDTO(String dataSourceKey) {
+    public DataSourceUnsecuredDTO getDataSourceDTO(String dataSourceKey) throws IOException {
 
         Source source = getByKey(dataSourceKey);
-        return conversionService.convert(source, DataSourceUnsecuredDTO.class);
+        return dataSourceMapper.toDatasourceUnsecuredDTO(source);
+    }
+
+    public <T> T executeOnSource(DataSourceUnsecuredDTO dataSourceData, JdbcTemplateConsumer<T> consumer) throws IOException {
```
Comment (pavgra) [inline service/src/main/java/org/ohdsi/gisservice/service/SourceService.java]: What about re-usage?

**Review by pavgra — CHANGES_REQUESTED** [review]

File: service/src/main/java/org/ohdsi/gisservice/config/EncryptionConfig.java (hunk 1 above)
Comment (wivern) [reply service/src/main/java/org/ohdsi/gisservice/config/EncryptionConfig.java:22]: Sure, I guess the best candidate to move to is arachne-commons:datasource-manager.

**Review by pavgra — APPROVED** [review]

**Pull request merged**